composer.json

```diff
@@ -14,11 +14,11 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^7.2 || ^8.0",
     "laravel/framework": "^6.0",
     "zircote/swagger-php": "~2.0|3.*",
     "swagger-api/swagger-ui": "^3.0",
-    "symfony/yaml": "^4.1"
+    "symfony/yaml": "^5.0"
```

Comment:
Maybe we can do `>=4.1` or `^4.1 || ^5.0`?

Reply:
Yes we van! ;-)